### DISCUSSION TITLE
iSSG - fallback: true for non-dynamic route pages

### DISCUSSION THREAD

**Discussion by donferi** [vercel/next.js#23315] [Help]
Is there a way to do a `fallback: true` for non-dynamic route pages?

**Here is my use case:**
I'm trying to use `iSSG` in a different way, almost like `SSR`. Imagine I have an app with 100s of non-dynamic route pages. I  want to build the static version of maybe only 10 pages at build time. All other pages would get built as requests arrive. Ideally I could use `fallback: true` and `revalidate`, however, since the pages are not dynamically routed I can't use `getStaticPaths` which means I can't use `fallback`.

Does this use-case makes sense? Am I missing something?

**Comment by wwwdepot** [comment]
Give a little more context? What are you trying to achieve?

**Reply by donferi** [reply]
Basically I just want to do iSSG with revalidation for non-dynamically routed pages.
